packages/edit-post/src/components/sidebar/settings-header/style.scss

```diff
@@ -41,8 +27,8 @@
 	&.is-active {
 		// The transparent shadow ensures no jumpiness when focus animates on an active tab.
 		box-shadow: inset 0 0 0 $border-width-focus transparent, inset 0 0 -$border-width-tab 0 0 var(--wp-admin-theme-color);
-		font-weight: 600;
 		position: relative;
+		z-index: z-index(".edit-post-sidebar__panel-tab.is-active");
```

Comment:
Why do we need a z-index?

Reply:
<img width="208" alt="Screenshot 2020-06-16 at 15 09 34" src="https://user-images.githubusercontent.com/1204802/84778530-6844d580-afe3-11ea-97ea-2ca7cb3b96fa.png">

It makes the tab border more visible